CHANGELOG.asciidoc

```diff
@@ -111,6 +111,7 @@ https://github.com/elastic/beats/compare/v5.1.1...master[Check the HEAD diff]
 - Fix error on importing dashboards due to colons in the Caassandra dashboard. {issue}3140[3140]
 
 *Filebeat*
+- Config option `filebeat.config_dir` is deprecated. It is planned to replace it with a new implementation.
```

Comment:
SGTM. I'm confident enough in https://github.com/elastic/beats/issues/3430 to get it in for 6.0